client/css/style.css

```diff
@@ -893,6 +920,30 @@ kbd {
 	touch-action: pan-y;
 }
 
+#sidebar .join_form {
+	display: none;
+}
+
+#sidebar .join_form .input {
+	border-radius: 2px;
+	padding: 5px 4px;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 5px;
+	margin-bottom: 5px;
+	width: 80%;
+	border: 1px solid #ddd;
+}
+
+#sidebar .join_form .btn {
+	border-radius: 2px;
+	display: block;
+	width: 80%;
+	padding: 1px;
+	margin: auto;
+}
```

Comment:
Can you add `height: 29px` (height of `.input`)?

Reply:
done